platform/vs/docker-sonic-vs/zero_profiles.json

```diff
@@ -43,7 +43,7 @@
         "BUFFER_PROFILE_TABLE:egress_lossless_zero_profile" : {
             "pool":"egress_lossless_pool",
             "size":"0",
-            "dynamic_th":"-8"
+            "static_th":"0"
```

Comment:
don't see egress_lossless_zero_pool definition in this file

Reply:
No. There is no egress_lossless_zero_pool.
For egress, there is the egress_lossless_pool whose mode is static.
egress_lossless_zero_profile is the profile on top of the pool so its model should be aligned with the pool.

Reply:
In VS image, platform `x86_64-dell_s6000_s1220-r0` and HWSKU `Force10-S6000` is leveraged. The `egress_lossless_pool` is defined [here](https://github.com/Azure/sonic-buildimage/blob/master/device/dell/x86_64-dell_s6000_s1220-r0/Force10-S6000/buffers_defaults_t0.j2#L20) 
```
        "egress_lossless_pool": {
            "size": "12766208",
            "type": "egress",
            "mode": "static"
        },
```